package.json

```diff
@@ -33,7 +33,7 @@
     "commander":                        "^2.4.0",
     "cliff":                            "^0.1.10",
     "browserify":                       "^9.0.3",
-    "mocha-phantomjs":                  "^3.5.3",
+    "mocha-phantomjs":                  "3.5.3",
```

Comment:
Ah neat, I see this fixed the `Peer mocha-phantomjs@3.6.0 wants phantomjs@1.9.1 - 1.9.7-15` error we previously had. :+1: